README.md

```diff
@@ -469,7 +469,7 @@ include escaping code in it.
 
 In order to create a formatter instance, create a class that extends the
 [Formatter](https://github.com/ilib-js/i18nlint-common/blob/main/src/Formatter.js)
-class in the [i18nlint-common](https://github.com/ilib-js/i18nlint-common/] project.
+class in the [i18nlint-common](https://github.com/ilib-js/i18nlint-common/) project.
```

Comment:
should be i18nlint-common -> ilib-lint-common in all occurances in this file

Reply:
I fixed them all